pyccl/constants.py

```diff
@@ -1,20 +1,11 @@
-"""This file exposes constants present in CCL.
-
-"""
-from pyccl.ccllib import \
-  CCL_ClTracer, CCL_ClTracer_swigregister, \
-  CCL_ERROR_CLASS, CCL_ERROR_INCONSISTENT, CCL_ERROR_INTEG, CCL_ERROR_LINSPACE, \
-  CCL_ERROR_MEMORY, CCL_ERROR_ROOT, CCL_ERROR_SPLINE, CCL_ERROR_SPLINE_EV, \
-  CLIGHT_HMPC, \
-  CL_TRACER_NC, CL_TRACER_WL, CL_TRACER_CL, \
-  CCL_NONLIMBER_METHOD_NATIVE, CCL_NONLIMBER_METHOD_ANGPOW, \
-  CCL_CLT_NZ, CCL_CLT_BZ, CCL_CLT_SZ, CCL_CLT_WM, \
-  CCL_CLT_RF, CCL_CLT_BA, CCL_CLT_WL, \
-  DNDZ_NC, DNDZ_WL_CONS, DNDZ_WL_FID, DNDZ_WL_OPT, \
-  EPS_SCALEFAC_GROWTH, \
-  GNEWT, \
-  K_PIVOT, \
-  MPC_TO_METER, PC_TO_METER, RHO_CRITICAL, SOLAR_MASS, \
-  SplPar, SplPar_swigregister, \
-  Z_MAX_SOURCES, Z_MIN_SOURCES, \
-  CCL_CORR_FFTLOG, CCL_CORR_BESSEL, CCL_CORR_LGNDRE, CCL_CORR_GG, CCL_CORR_GL, CCL_CORR_LP, CCL_CORR_LM
+"""This file exposes constants present in CCL."""
+from .ccllib import (  # flake8: noqa
```

Comment:
Why have some imports gone away here?

Reply:
Mike suggested four to remove in CR. They are not used elsewhere.